db/seeds.rb

```diff
@@ -5,3 +5,18 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+spots = [
+  { name: 'Little Ducklings',location:'Boston Common, Boston MA',description: 'all the ducks', photo: 'https://cdn1.sph.harvard.edu/wp-content/uploads/sites/48/2012/09/make-way-for-ducklings-5.jpg' },
+  { name: 'Abe Blinkin',location:'1 Cambridge Sq, Cambridge MA',description: 'abe is the man', photo: 'http://www.talkingstatues.co.uk/stockist_images/chicagoadmin/530x1530_lincoln.jpg' },
+  { name: 'Little Ducklings',location:'Boston Common, Boston MA',description: 'all the ducks' },
+  { name: 'George Washington',location:'1 Cambridge Sq, Cambridge MA',description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum', photo: 'https://upload.wikimedia.org/wikipedia/commons/9/92/George_Washington_statue_in_the_Boston_Public_Garden_-_DSC08201.JPG' },
+  { name: 'Sam Adams',location:'Boston Common, Boston MA',description: 'great beer' },
+  { name: 'Lorem ipsum',location:'1 Cambridge Sq, Cambridge MA',description: 'abe is the man', photo: 'https://cdn-www.enfocus.com/sites/combell-www.enfocus.com/files/media/blog/2017-08-09-Lorem-Ipsum/lorem-ipsum.jpg' },
+  { name: 'Christian the Pirate',location:'Deck 7 Common, Boston MA',description: 'fastest eyes' },
+  { name: 'Evan the Great',location:'1 Cambridge Sq, Cambridge MA',description: 'lorem ipsum dolum' }
+]
+
+spots.each do |item|
+  spot = Spot.create(item)
```

Comment:
since this is a script we should `create!` using the `!` variant will ensure we get an exception if validation fails.